src/webhooks/types.ts

```diff
@@ -8,6 +8,7 @@ type WebhookHandlerFunction = (
   topic: string,
   shop_domain: string,
   body: string,
+  webhookId: string,
```

Comment:
I think we should probably make this optional to not break existing apps that are using TS.